ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/TransactionsReader.scala

```diff
@@ -97,12 +97,12 @@ private[appendonlydao] final class TransactionsReader(
     val query = (range: EventsRange[(Offset, Long)]) => {
       implicit connection: Connection =>
         logger.debug(s"getFlatTransactions query($range)")
-        QueryNonPruned.executeSqlOrThrow(
-          EventsTableFlatEvents
-            .preparePagedGetFlatTransactions(sqlFunctions)(
-              range = EventsRange(range.startExclusive._2, range.endInclusive._2),
-              filter = filter,
-              pageSize = pageSize,
+        queryNonPruned.executeSqlOrThrow(
+          new EventsTableFlatEventsRangeQueries.GetTransactions(storageBackend)
```

Comment:
good point, this was the behavior before as well, but since you mentioned it, I will put that instantiation on the TransactionReader as with the eventSeqIdReader